adapters/decenterads/decenterads.go

```diff
@@ -0,0 +1,155 @@
+package decenterads
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type DecenterAdsAdapter struct {
```

Comment:
Please name this `adapter`. There is no need to repeat the package name and there is no reason to export it (so it should start with a lower case letter- isn't Go access modifiers fun?).